MyTest/src/test/java/com/test/ui/BlouseTest.java

```diff
@@ -0,0 +1,36 @@
+package com.test.ui;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.*;
+import pages.*;
+import static Data.TestData.*;
+
+public class BlouseTest extends SetTheBrowser{
+    @BeforeClass
+    public void beforeClass(){
+        openBrowser();
+    }
+    @Test()
+    public void blouseTest(){
+
+        SearchPage srchPg = PageFactory.initElements(driver, SearchPage.class);
```

Comment:
It can be moved to class constructor PageFactory.initElements(driver, SearchPage.class); instead of calling it on test level